test/integration/h2_capture_fuzz.proto

```diff
@@ -0,0 +1,158 @@
+syntax = "proto3";
+
+package test.integration;
+
+message H2FramePing {
+  enum Flags {
+    None = 0;
+    Ack = 1;
+  }
+  Flags flags = 1;
+  bytes data = 2;
+}
+
+message H2FrameSettings {
+  enum Flags {
+    None = 0;
+    Ack = 1;
+  }
+  Flags flags = 1;
+}
+
+enum HeadersFlags {
+  None = 0;
+  EndStream = 1;
+  EndHeaders = 4;
+}
+
+message H2FrameHeaders {
+  HeadersFlags flags = 1;
```

Comment:
Are any of these flags possible to appear together? Should this be a `repeated`?

Reply:
+1 that, or HeadersFlag should have an EndStreamAndHeaders option

Reply:
Made the headers flag field `repeated`.